django-data/image/validation/tests/test_tasks.py

```diff
@@ -91,18 +101,45 @@ def setUp(self):
 
         # another patch
         self.send_msg_ws_patcher = patch(
-                'validation.tasks.send_message_to_websocket')
+            'validation.tasks.send_message_to_websocket')
         self.send_msg_ws = self.send_msg_ws_patcher.start()
 
-    def check_async_called(self, message, notification_message, pk=1):
+        # mocking validation summary
```

Comment:
Same as websocket mocking: there's always a patcher that need to be started to have a mock object to test. You have to stop the patcher in order to test `call_count` for instance. Setting in `setUp` means that this code is called before calling a generic `test_` method, and is called once for each `test_` method. `setUpClass` to setup method once when class is called during test, and the same instance is used in every test.

Reply:
👍